docs/reference/helm-config.txt

```diff
@@ -152,12 +152,27 @@ Helm Chart </helm/determined-helm-chart.tgz>`.
       use during distributed training. A valid port range is in the
       format of ``MIN:MAX``.
 
+   -  ``forcePullImage``: Defines the default policy for forcibly pulling images from
+      the docker registry and bypassing the docker cache. If a pull policy is specified
+      in the :ref:`experiment config <exp-environment-image>` this default  is overriden.
+      Defaults to ``false``.
```

Comment:
update this to include the detail about docker pulls as well?

Reply:
done